editor/store/selectors.js

```diff
@@ -444,17 +444,6 @@ export function getDocumentTitle( state ) {
 	return title;
 }
 
-/**
- * Returns a URL to preview the post being edited.
- *
- * @param {Object} state Global application state.
- *
- * @return {string} Preview URL.
- */
-export function getEditedPostPreviewLink( state ) {
```

Comment:
Should we deprecate this? On the one hand, it's a public 'method'. On the other, it's undocumented.

Reply:
It's not documented, nothing else uses it, and @aduth mentioned it just missed the cutting block during another commit. I think technically maybe we should but I feel like we're in the clear.

I will add it back, deprecate it, and wear the dunce hat if it affects anyone in the next release 😉

Reply:
Worth noting: It doesn't work as-is anyways, hence the original issue. Probably better to not work than to throw errors though. If we wanted full compatibility, we might redirect the result to `getAutosaveAttribute( 'preview_link' )`

Reply:
But `getAutosaveAttribute( 'preview_link' )` won't work on page load, right? I tested this with new page loads of existing posts and "Add New" posts and it never broke the page 🤷‍♂️